test/packages/apache/docs/README.md

```diff
@@ -218,7 +218,6 @@ An example event for `status` looks as following:
 | apache.status.cpu.load | CPU Load. | scaled_float |
 | apache.status.cpu.system | System cpu. | scaled_float |
 | apache.status.cpu.user | CPU user load. | scaled_float |
-| apache.status.hostname | Apache hostname. | keyword |
```

Comment:
Is this change intentional? At first glance it seems unrelated to this PR.

Reply:
It's related, because it's the first time the README has been rendered correctly :)